### DISCUSSION TITLE
Hyper-personal spam

### DISCUSSION THREAD

**Discussion by TonyGravagno** [rspamd/rspamd#3723] [Q&A]
There is an entire category of spam that marketers call "hyper personalization" - Rspamd isn't catching it.
https://www.google.com/search?q=hyper+personalization+marketing+spam

Examples:
- "Hey Tony, how does Thursday at 1:00 PM work for a meeting?"
- "I'm trying to get in touch with you to schedule a brief phone call..."
- "Maybe we should finally talk :)"
- "Hope you have had a great weekend. Wanted to try and catch you regarding my last email. Do you have some time to discuss today?"
- "Hope you’ve had a great start to your week! I really wanted to get in touch, as I came across your organization online."
- "You've probably been busy or this just hasn't been a priority- I get it and no problem. That said, ..."
- "This a quick reminder that in just a couple of days' time, Wendy Weiss is going to show you, in real time, how to get your sales team..."

All of this is pure, automated, Unsolicited Bulk Email = Spam. It's written to appear as though there has been a prior contact.

This gets by greylisting. DKIM,SPF, DMARC all pass.
Junk folders for spam/ham training don't seem to help. (Could be misconfiguration on my part.)
X-Spamd-Results are from about -2 to +2.
PREVIOUSLY_DELIVERED shows as zero, probably because they just keep sending. But I would think that when mail is only in the Junk folder that it should't be categorized as PREVIOUSLY_DELIVERED.

I'm not getting any of this in Gmail, so Google seems to have a working solution.
 
What can we do to improve on this? I have the luxury of being able to tweek settings and install beta tests on a production server that is only used for my own domains. I'm happy to test solutions.

Thanks!

**Comment by TomFuch** [comment]
Do you have DCC Module enabled?

**Reply by TonyGravagno** [reply]
I do not. I'll do my homework on that, see how well it addresses this scenario and report back with details
Thanks @TomFuch !
